File: Projects/server/Contracts/Responses/TripDto.cs
namespace server.Contracts.Responses
{
    public class TripDto
    {
        public ulong TripId { get; set; }
        public DateTime StartDatetime { get; set; }
        public DateTime? EndDatetime { get; set; }
        public uint DeviceId { get; set; }
        public CarDto Car { get; set; } = null!;
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/responses/TripDto.kt
package com.example.dataproviderapp.dto.responses

import java.time.LocalDateTime

data class TripDto(
    val tripId: ULong,
    val startDatetime: LocalDateTime,
    val endDatetime: LocalDateTime?,
    val deviceId: UInt,
    val car: CarDto,
)
